src/utils/codeowners.ts

```diff
@@ -22,7 +22,7 @@ export type ownerRule = {
 };
 
 const filterGeneratedContent = (content: string): [withoutGeneratedCode: string[], blockPosition: number] => {
-  const lines = content.split('\n');
+  const lines = content.split(/\r?\n/);
```

Comment:
nit: I will move this to a const and add a tiny explanation to it.

Reply:
Done! Feel free to take a look if the comment makes sense. 🙂